src/OpenTelemetry.Instrumentation.StackExchangeRedis/StackExchangeRedisCallsInstrumentationOptions.cs

```diff
@@ -28,5 +31,15 @@ public class StackExchangeRedisCallsInstrumentationOptions
         /// Gets or sets the maximum time that should elapse between flushing the internal buffer of Redis profiling sessions and creating <see cref="Activity"/> objects. Default value: 00:00:10.
         /// </summary>
         public TimeSpan FlushInterval { get; set; } = TimeSpan.FromSeconds(10);
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the <see cref="StackExchangeRedisCallsInstrumentation"/> should use reflection to get more detailed <see cref="SemanticConventions.AttributeDbStatement"/> tag values. Default value: False.
+        /// </summary>
+        public bool SetVerboseDatabaseStatements { get; set; }
+
+        /// <summary>
+        /// Gets or sets.
+        /// </summary>
+        public Action<Activity, IProfiledCommand> Enrich { get; set; }
```

Comment:
could you update the readme.md as well to reflect new features?
Also the doc comment for this seems incomplete.

Reply:
Done